packages/jobs/lib/execution/sync.ts

```diff
@@ -385,6 +387,7 @@ export async function handleSyncSuccess({ nangoProps }: { nangoProps: NangoProps
             runTime: (new Date().getTime() - nangoProps.startedAt.getTime()) / 1000,
             failureSource: ErrorSourceEnum.CUSTOMER,
             isCancel: false,
+            version: nangoProps.syncConfig.version,
```

Comment:
at this stage, maybe we should just pass `nangoProps` instead

Reply:
ideally yes but there are paths where things failed before the nangoProps is instanciated (ie: exception thrown early in startSync)